tests/UnitTests/CoreGraphics.drawing/EntryPoint.cpp

```diff
@@ -82,6 +82,11 @@ class CommandLineDrawingTestConfigImpl : public DrawingTestConfigImpl {
     virtual std::string GetComparisonPath() override {
         return _comparisonPath;
     }
+
+    virtual std::string GetResourcePath(const std::string& resource) override {
+        // Using / as a path separator is valid on both Windows and OS X.
+        return __ModuleDirectory() + "/data/" + resource;
```

Comment:
for some reason I was thinking it wasn't. Double checked and it is.
gotcha.

---
In reply to: [90145289](https://github.com/Microsoft/WinObjC/pull/1470#discussion_r90145289) [](ancestors = 90145289)